docs/README.md

```diff
@@ -1,4 +1,125 @@
+   * Convert an anonymous type to a tuple.
+   * For a foreach loop to LINQ query or to LINQ method.
+   * Generate deconstruct method
+* Categorize references by Read/Write in Find All References window.
+* Add Editorconfig when_multiline option for csharp_prefer_braces.
+* New classification colors are available from the .NET Compiler Platform SDK (aka Roslyn). New default colors, similar to the Visual Studio Code colors, are gradually being rolled out. You can adjust these colors in Tools > Options > Environment > Fonts and Colors or turn them off in Environment > Preview Features by unchecking the Use enhanced colors check box. We’d appreciate hearing feedback on how this change affects your workflow.
+* Configure [code cleanup](https://docs.microsoft.com/en-us/visualstudio/ide/code-styles-and-code-cleanup) (settings formally a part of Format Document)
+* [Format document global tool](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes#-apply-code-style-preferences) can be run from the command-line
+ * Use [code metrics](https://docs.microsoft.com/visualstudio/code-quality/code-metrics-values) with .NET Core projects with our added compatibility.
+ * Export editor settings to an Editorconfig file through **Tools > Options > Text Editor > C# > Code Style** with the button "Generate .editorconfig file from settings".
+ * Use C# and Visual Basic's new Regex parser support. Regular expressions are now recognized, and language features are enabled on them. Regex strings are either recognized when a string is passed to the Regex constructor or when a string is immediately preceded with a comment containing the string language=regex. The language features included in this release are classification, brace matching, highlight references, and diagnostics.
+ * You can now use dead code analysis for unused private members with an optional code fix to remove unused member declaration.
+ * The Find References feature on an accessor now only returns results for that accessor.
+ * "Using" statements can be added when code is pasted into a file. A code fix appears after pasting recognized code that prompts you to add relevant missing imports.
+ * You can now use Find All References (Shift-F12) and CodeLens to show results from Razor (.cshtml) files in .NET Core projects. You can then navigate to the identified code in the relevant Razor files.
+ * You will now receive a warning when running code analysis using FxCop. .NET Compiler analyzers are the recommended way to perform code analysis going forward. Read more on migrating to .NET compiler platform analyzers.
+
+## <a id="prod-15"></a> dev15
+
+### dev15 Summary
+* Visual Studio-based code cleanup via Format Document 
+* Refactorings: InvertIf, add parameter from method callsite, remove unnecessary parentheses, convert if-else assignment, completion for attributes and return to ternary conditional. 
+* Sourcelink support for GitLab, GitHub Enterprise and On-Premise TFS  
```

Comment:
I don't think SourceLink should be listed here. It has nothing to do with Roslyn.

Reply:
Sure, I can take it out.